webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPage.js

```diff
@@ -4,44 +4,44 @@ import { Button, Icon } from 'patternfly-react';
 import './auditspage.scss';
 
 import { translate as __ } from '../../../common/I18n';
+import { getManualURL, newWindowOnClick } from '../../../common/helpers';
 import PageLayout from '../../common/PageLayout/PageLayout';
 import AuditsTable from './components/AuditsTable';
-import { AUDITS_SEARCH_PROPS, AUDITS_MANUAL_URL } from '../constants';
-import { useForemanVersion } from '../../../Root/Context/ForemanContext';
+import { AUDITS_SEARCH_PROPS } from '../constants';
 
 const AuditsPage = ({
   searchQuery,
   fetchAndPush,
   isLoading,
   hasData,
   ...props
-}) => {
-  const foremanVersion = useForemanVersion();
-  return (
-    <PageLayout
-      header={__('Audits')}
-      searchable
-      searchProps={AUDITS_SEARCH_PROPS}
-      searchQuery={searchQuery}
-      isLoading={isLoading && hasData}
-      onSearch={search => fetchAndPush({ searchQuery: search, page: 1 })}
-      onBookmarkClick={search => fetchAndPush({ searchQuery: search, page: 1 })}
-      toolbarButtons={
-        <Button href={AUDITS_MANUAL_URL(foremanVersion)} className="btn-docs">
-          <Icon type="pf" name="help" />
-          {__(' Documentation')}
-        </Button>
-      }
-    >
-      <AuditsTable
-        fetchAndPush={fetchAndPush}
-        isLoading={isLoading}
-        hasData={hasData}
-        {...props}
-      />
-    </PageLayout>
-  );
-};
+}) => (
+  <PageLayout
+    header={__('Audits')}
+    searchable
+    searchProps={AUDITS_SEARCH_PROPS}
+    searchQuery={searchQuery}
+    isLoading={isLoading && hasData}
+    onSearch={search => fetchAndPush({ searchQuery: search, page: 1 })}
+    onBookmarkClick={search => fetchAndPush({ searchQuery: search, page: 1 })}
+    toolbarButtons={
+      <Button
+        onClick={newWindowOnClick(getManualURL('4.1.4Auditing'))}
```

Comment:
did you try passing `target="_blank"` directly to the component? First look at its implementation looks like it could work.

Reply:
Good point, updated.